### ISSUE TITLE
Problem With Openelec OR Libreelec LiveTV

### ISSUE THREAD

**Issue by enrydgl** [maxnet/berryboot#329]
Hi,
 there's a problem with firmware DVB-T to activate TvHeadend Add-On on Openelec or Libreelec in BerryBoot.
With ONLY OPENELEC SO my usb dvb-t work good. With Dmesg SSH:
`[    9.357865] usb 1-1.5: dvb_usb_af9035: prechip_version=00 chip_version=03 chip_type=3802
[    9.358277] usb 1-1.5: dvb_usb_v2: found a 'AVerMedia AVerTV Volar HD/PRO (A835)' in cold state
[    9.359532] usb 1-1.5: Direct firmware load for dvb-usb-af9035-02.fw failed with error -2
[    9.359567] usb 1-1.5: dvb_usb_v2: Did not find the firmware file 'dvb-usb-af9035-02.fw'. Please see linux/Documentation/dvb/ for more details on firmware-problems. Status -2
[    9.359622] dvb_usb_af9035: probe of 1-1.5:1.0 failed with error -2
[    9.359783] usbcore: registered new interface driver dvb_usb_af9035
[   12.043953] random: nonblocking pool is initialized`

When I try to download "dvb-usb-af9035-02.fw" driver file on directory lib/firmware, wget give me "file exist". 
I try to remove dvb-usb-af9035-02.fw file and download another time the driver file. After reboot the problem DMESG is same. DVB-T don't mount and don't work! Why?

Thanks.

**Comment by maxnet** [comment]
Does it work if you retry loading the driver later?

```
rmmod dvb_usb_af9035
modprobe dvb_usb_af9035
```

**Comment by enrydgl** [comment]
Yes if I retry loading driver work good. Can you give me a method to reloading driver during start Openelec or Librelec distro?

**Comment by maxnet** [comment]
Might be easer to copy the firmware file to the /shared/lib/modules folder on the Berryboot ext4 parition.

**Comment by enrydgl** [comment]
Excuse me... but I'm very newbie with Linux. Can you write istructions to save firmware file in /shared/lib/modules directory?

**Comment by enrydgl** [comment]
Ok I try to copy firmware in /shared/lib/modules.. but doesn't work. 
To solve it (but it isn't a right solution), I create a script file (.sh) to generate and download a week EPG TV programs and in this script I add the commands to remount dvb-t driver.

The script start when TvHeadend backend add-on start in librelec or openelec (EPG setting in TvHeadend backend addon).
